index.html

```diff
@@ -322,7 +332,7 @@ <h2>Additional values for the <code>Prefer</code> header</h2>
             </li>
             <li>
               Implementations MAY support <code>http://www.w3.org/ns/oa#PreferContainedDescriptions</code>
-              to request that the response include representations of any <a>LDPC</a> contained resources in the
+              to request that the response include representations of any <a>LDPC</a>-contained resources in the
```

Comment:
s/include/includes

Reply:
I like "include" better here — though maybe it should be "that the response should include"?

Reply:
Sure enough, it's `he/she/it includes`, but `he/she/it include` if it's subjunctive, which I guess this is.  I stand corrected.